docs/management_api.yml

```diff
@@ -472,10 +472,9 @@ definitions:
     example:
       application/json:
```

Comment:
@mchalski that seems to be required

Reply:
oh, oh... I can see it now. approved - if you're sure it renders ok .